controller/pagecontroller.php

```diff
@@ -85,7 +85,9 @@ public function index() {
 		$params = [
 			'appName' => $appName,
 			'uploadUrl' => $this->urlGenerator->linkTo('files', 'ajax/upload.php'),
-			'publicUploadEnabled' => $this->appConfig->getAppValue('core', 'shareapi_allow_public_upload', 'yes')
+			'publicUploadEnabled' => $this->appConfig->getAppValue('core', 'shareapi_allow_public_upload', 'yes'),
```

Comment:
@imjalpreet - 100 characters max, otherwise I get reports like these:
https://scrutinizer-ci.com/g/interfasys/galleryplus/inspections/b19f7e2a-00e4-42b3-b6e9-7aabc3e7899e/code-structure/operation/OCA%5CGalleryPlus%5CController%5CPageController%3A%3Aindex